include/zephyr/net/wifi_mgmt.h

```diff
@@ -48,6 +48,13 @@ extern "C" {
 #endif /* CONFIG_WIFI_MGMT_SCAN_CHAN_MAX_MANUAL */
 
 #define WIFI_MGMT_BAND_STR_SIZE_MAX 8
+#define WIFI_MGMT_SCAN_MIN_DWELL_TIME_ACTIVE 5
```

Comment:
some references for the numbers would be good? I remember we had them in the Kconfig before they were removed.